src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <div class="card__container">
+        <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="card__image"
+        >
+        <h1 class="card__item">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h3 class="card__code">
+          Product code: 195434
+        </h3>
+        <div class="stars">
+          <div class="stars__container">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__stars"></div>
+          </div>
+          <h3 class="stars__rewiew">Reviews: 5</h3>
+        </div>
+        <div class="card__price">
+          <h3 class="card__price price-item">
+            Price:
+          </h3>
+          <h2 class="card__price price">
+            $2,199
+          </h2>
+        </div>
+        <a
```

Comment:
```suggestion
        <a
          href="#"
          data-qa="hover"
          class="card__button"
        >
          BUY
        </a>
```

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3458/files#r1287178973
didn't change it

Reply:
isn't fixed